### PR TITLE
#63 Implement front-end for "Budget Report" page

### PR DESCRIPTION
**Pull request by Ziadsharkos**
Implemented front-end for "Budget Report" page: Page displaying the info about the condo fee revenue, operation costs, and profits (for now generates random fees, and costs, then calculates the profit and total)

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -38,6 +45,11 @@ const routes: Routes = [
     component: BuildingInfoComponent,
     canActivate: [AuthguardGuard],
   },
+  {
+    path: 'locker',
+    component: LockerComponent,
```
Comment (Guillaume-Lachapelle) [inline src/app/app-routing.module.ts:50]: Why was this route added?
The rest looks good

**Review by Guillaume-Lachapelle — APPROVED** [review]

**Review by IsabelleCzuboka — APPROVED** [review]
Looks good to me

**Review by amczuboka — APPROVED** [review]
LGTM

**Review by npiperni — APPROVED** [review]
Looks good, just also wondering why the lockers path was added

**Review by KarinaSandur — APPROVED** [review]

**Review by CarlaShawi — APPROVED** [review]

**Review by Tharsipa — APPROVED** [review]

**Review by Holyvert — APPROVED** [review]

**Pull request merged**